drivers/ethernet/CMakeLists.txt

```diff
@@ -27,7 +27,6 @@ zephyr_library_sources_ifdef(CONFIG_ETH_STELLARIS	eth_stellaris.c)
 zephyr_library_sources_ifdef(CONFIG_ETH_STM32_HAL	eth_stm32_hal.c)
 zephyr_library_sources_ifdef(CONFIG_ETH_W5500		eth_w5500.c)
 zephyr_library_sources_ifdef(CONFIG_ETH_SAM_GMAC	eth_sam_gmac.c)
-zephyr_library_sources_ifdef(CONFIG_DSA_KSZ8XXX		dsa_ksz8xxx.c)
```

Comment:
Why is this deleted?

Reply:
It's a duplicate. It's in the commit log.